modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -16121,6 +16129,84 @@ private void _testGetObjectEntriesFilteredBySystemDate(String fieldName)
 		_assertFilteredObjectEntries(2, fieldName + " ne null");
 	}
 
+	private void _testGetObjectEntriesPageWithObjectActionsScope(
```

Comment:
Please rename this method name to just `_testGetObjectEntriesPageWithObjectActions(`